manual/azure-sru-19.3.41.txt

```diff
@@ -83,64 +126,83 @@ for series in $SRU_SERIES; do
         fi
         az vm create --name=$name \
             --image=Canonical:UbuntuServer:$image_version:latest \
             --admin-username=root --admin-username=ubuntu \
             -g $RESOURCE_GROUP --ssh-key-value $SSH_KEY \
-            $BOOT_DIAG \
+            $BOOT_DIAG_ARG \
             --custom-data sethostname.yaml $nics
         IP=$(az vm list-ip-addresses --name $name | jq -r '.[] | .virtualMachine.network.publicIpAddresses[].ipAddress'| awk '{printf "ubuntu@%s", $1}')
 
-        ssh "${sshopts[@]}" $IP -- cloud-init status --wait --long
-        ssh "${sshopts[@]}" $IP -- dpkg-query --show cloud-init
-        ssh "${sshopts[@]}" $IP -- sudo cat /run/cloud-init/result.json
-        ssh "${sshopts[@]}" $IP -- grep Trace /var/log/cloud-init.log
-        ssh "${sshopts[@]}" $IP -- systemd-analyze
-        ssh "${sshopts[@]}" $IP -- systemd-analyze blame
-        ssh "${sshopts[@]}" $IP -- cloud-init analyze show
-        ssh "${sshopts[@]}" $IP -- cloud-init analyze blame
-        echo 'Networking config'
-        ssh "${sshopts[@]}" $IP -- cat $netcfg
+    echo "Created $name: ubuntu@$IP"
 
-        scp "${sshopts[@]}" setup_proposed.sh $IP:.
-        ssh "${sshopts[@]}" $IP -- sudo bash ./setup_proposed.sh 2>&1 | egrep 'cloud-init'
-        ssh "${sshopts[@]}" $IP -- dpkg-query --show cloud-init
-        ssh "${sshopts[@]}" $IP -- sudo hostname SRU-didnt-work
-        ssh "${sshopts[@]}" $IP -- sudo rm -f $netcfg
-        ssh "${sshopts[@]}" $IP -- sudo cloud-init clean --logs --reboot
+        ssh $sshopts $IP -- cloud-init status --wait --long
+        ssh $sshopts $IP -- dpkg-query --show cloud-init
+        ssh $sshopts $IP -- sudo cat /run/cloud-init/result.json
+        ssh $sshopts $IP -- grep Trace /var/log/cloud-init.log
+        ssh $sshopts $IP -- systemd-analyze
+        ssh $sshopts $IP -- systemd-analyze blame
+        ssh $sshopts $IP -- cloud-init analyze show
+        ssh $sshopts $IP -- cloud-init analyze blame
+        echo 'Networking config'
+        ssh $sshopts $IP -- cat $netcfg
 
+        scp $sshopts setup_proposed.sh $IP:.
+        ssh $sshopts $IP -- sudo bash ./setup_proposed.sh 2>&1 | egrep 'cloud-init'
+        ssh $sshopts $IP -- dpkg-query --show cloud-init
+        ssh $sshopts $IP -- sudo hostname SRU-didnt-work
+        ssh $sshopts $IP -- sudo rm -f $netcfg
+        if [ "" = "$nics" ]; then
+            ssh $sshopts $IP -- sudo cloud-init clean --logs --reboot
+        else
+        echo "Shutting down instance and adding IPv6 config to nic"
+        az vm stop --name=$name -g $RESOURCE_GROUP 
+            az network nic ip-config create \
+        --name $NIC_NAME-$series-IPv6Config \
+        --nic-name $NIC_NAME-$series \
+                --resource-group $RESOURCE_GROUP \
+                --vnet-name $VNET_NAME \
+                --subnet sruSubnet \
+        --private-ip-address-version IPv6
+        az vm start --name=$name -g $RESOURCE_GROUP 
```

Comment:
This block needs indenting under the else clause.  Maybe it's tabs making it look wrong.  If so, drop the tabs please.